### PR TITLE
GEN-1059 | Create `QuickAdd` component for incomplete quotes

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

- Add container to represent an incomplete quick add card!

## Justify why they are needed

- This is an exploration to understand what kind of data we need from the API to represent an incomplete quick add card. Therefore a lot of things are missing or contain placeholder data.

- Not yet in use.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3167** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3167" target="_blank">Graphite</a> 
    * **PR #3169** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3169" target="_blank">Graphite</a> 
      * **PR #3170** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3170" target="_blank">Graphite</a> 
        * **PR #3171** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3171" target="_blank">Graphite</a> 
          * **PR #3172** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3172" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3172?utm_source=stack-comment).

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3167** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3167" target="_blank">Graphite</a> 
    * **PR #3169** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3169" target="_blank">Graphite</a> 
      * **PR #3170** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3170" target="_blank">Graphite</a> 
        * **PR #3171** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3171" target="_blank">Graphite</a> 
          * **PR #3172** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3172" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3172?utm_source=stack-comment).

File: apps/store/src/components/QuickAdd/QuickAddIncompleteContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+  const displayItems = useDisplayItems(props.priceIntent.data)
+
+  const editLink = new URL(props.priceIntent.product.pageLink, ORIGIN_URL)
+  editLink.searchParams.set(OPEN_PRICE_CALCULATOR_QUERY_PARAM, '1')
+  editLink.searchParams.set(PRELOADED_PRICE_INTENT_QUERY_PARAM, props.priceIntent.id)
+
+  type Offer = (typeof props.priceIntent.offers)[number]
+  const offer = props.priceIntent.offers[0] as Offer | undefined
+
+  const cost = offer
```
Comment (alebedev) [inline apps/store/src/components/QuickAdd/QuickAddIncompleteContainer.tsx:38]: Would extracting helper function improve readability?

**Review by alebedev — APPROVED** [review]

File: apps/store/src/components/QuickAdd/QuickAddIncompleteContainer.tsx (hunk 1 above)
Comment (robinandeer) [reply apps/store/src/components/QuickAdd/QuickAddIncompleteContainer.tsx:38]: yes probably, we could make it into some sort of utility function by this point

**Comment by robinandeer** [comment]
### Merge Activity

* **Sep 8, 3:03 PM**: @robinandeer started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3172).

**Pull request merged**